fleet_vehicle_configuration/models/fleet_configuration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Camptocamp (https://www.camptocamp.com).
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class FleetVehicleConfiguration(models.Model):
+    _name = "fleet.vehicle.configuration"
+    _description = "Manage vehicle configuration"
+
+    vehicle_id = fields.Many2one("fleet.vehicle")
+    name = fields.Char(required=True)
+    max_seats = fields.Integer(required=True, default=10)
```

Comment:
Maybe I'm wrong but I seem to recall you can't have required=True on numeric fields cause they're never `False` in psql
If this is true, then you need an `_sql_constraint`